src/vs/editor/browser/editorBrowser.ts

```diff
@@ -461,6 +461,11 @@ export interface ICodeEditor extends editorCommon.IEditor {
 	 * @event
 	 */
 	onMouseLeave(listener: (e: IPartialEditorMouseEvent) => void): IDisposable;
+	/**
+	 * An event emitted on a "mousewheel"
+	 * @event
```

Comment:
As we are going to move to `wheel`, let's not expose it publicly at this point. `@internal`.